### DISCUSSION TITLE
How to auto record Live Streams on Youtube

### DISCUSSION THREAD

**Discussion by tr0n64** [streamlink/streamlink#3624] [Q&A]
Hi streamlink community 
I want automatic record a Live Stream from Youtube when the channel goes Live
Is there any script or tool to do this 
thanks a lot

**Comment by gravyboat** [comment]
While this is possible it's not something we will provide support or help you do. You'll have to figure it out on your own.

**Reply by tr0n64** [reply]
Ok but maybe there is someone in the streamlink community maybe have a script or link to a tool.
Or is there any streamlink community board where i can ask for help?

**Comment by tr0n64** [comment]
Anyone has TIPP how to make this?

**Reply by WolfganP** [reply]
Did you try `--output` as per https://streamlink.github.io/cli.html ?
